### PR TITLE
Feat/add react form builder

### PR DESCRIPTION
**Pull request by robertoHeCi**
### Main changes

- Add react form builder library
- Include a menu to navigate over the different CMSs
- Add a form in the Strapi page using a data model coming from aia as example

### Screenshots

Screenshot 2024-01-23 at 09 40 37

### Context

- :ticket: Closes / Relates (GitHub ticket ref)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .DS_Store
+.idea
```
Comment (pabloelisseo) [inline .gitignore:2]: para estas cosillas hay que hacerse un [.gitignore general](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) heheh

File: next-app/app/strapi/page.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// @ts-ignore
+
+import {FormBuilder} from "@guidesmiths/react-form-builder"
+
+const StrapiPage = async () => {
+    const {form} = await getForm()
+    return (
+        <div className="flex items-center justify-center max-h-full h-screen">
+            <div className="flex">
+                {form && <FormBuilder form={form} onSubmit={() => console.log("sent")}/>}
```
Comment (pabloelisseo) [inline next-app/app/strapi/page.tsx:12]: Could you console.log the submitted data? Just to see that everything is coming through, nothing fancy.

File: .gitignore (hunk 1 above)
Comment (pabloelisseo) [reply .gitignore:2]: Feel free to ignore this comment

**Review by pabloelisseo — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (robertoHeCi) [reply .gitignore:2]: It's actually a general .gitignore xD

Comment (robertoHeCi) [reply .gitignore:2]: We can add it in further PRs  I guess

**Pull request merged**